.scrutinizer.yml

```diff
@@ -1,9 +1,30 @@
-tools:
-    external_code_coverage: true
+build:
```

Comment:
do not like the idea of running tests both on travis and scrutinizer... is necessary to configure two envs...
sending code coverage from travis to scrutinizer is a common practice. 
any reason for it? am I missing something?